client/src/components/Home.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import '../containers/MapContainer.css'
+
+
+export default function Home() {
```

Comment:
Although Steve wants to call our link "Home", can we name this component "Map"?  Conceivably in the future, the Home link might point at something else.  Also, I would also ask to rename the ".css" file "Map.css" since I guess "Container" is implied since it is in the "containers" folder.

Reply:
Done! Additionally, please let me know if Map.css belongs in the containers folder. I'm not sure if I placed that right or not.